src/include/simeng/memory/MMU.hh

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <functional>
+#include <memory>
+
+#include "simeng/memory/Mem.hh"
+
+typedef std::function<uint64_t(uint64_t, uint64_t)> VAddrTranslator;
+
+namespace simeng {
+namespace memory {
+
+typedef std::function<void(DataPacket)> sendResponseToCore;
```

Comment:
Whilst the renaming is good - I dont think it is entirely accurate? Doesn't the response go back to a memory interface?

Reply:
Isnt the memory interface registered with a core? I also think saying `sendResponseToCore` helps readability and clearly communicates intent. Ultimately the response does get sent back to core.

Reply:
But this function operates inside of a MemoryInterface, with the DataPacket being used to add an item to a vector member variable of the MemoryInterface; which the core will then access when appropriate. 

I understand your logic behind calling it sendResponseToCore, but given it isn't talking with the core it could be deceptive as to where this function actually goes to/responds to

Reply:
reworded